examples/gke-daemonset-raid-disks.yaml

```diff
@@ -26,7 +26,7 @@ spec:
       hostPID: true
       containers:
       - name: startup-script
-        image: gcr.io/google-containers/startup-script:v1
+        image: registry.k8s.io/startup-script:v1
```

Comment:
This is essentially an image with a shell that hasn't been patched in an incredibly long time given it was available at Google-containers and our tags are immutable.

you should almost certainly switch to a maintained image and keep it up to date

Reply:
I find startup-script:v2 was created in 2019-06-18, but it's still too old.